plasmapy/formulary/collisions.py

```diff
@@ -2066,31 +2105,28 @@ def coupling_parameter(
     return coulomb_energy / kinetic_energy
 
 
-class CollisionFrequencies:
+class SingleParticleCollisionFrequencies:
     @particles.particle_input
     @validate_quantities(
-        v_a={"none_shall_pass": True, "can_be_negative": False},
-        T_a={
-            "none_shall_pass": True,
+        v_drift={"can_be_negative": False},
+        T_b={
             "can_be_negative": False,
             "equivalencies": u.temperature_energy(),
         },
         n_b={"can_be_negative": False},
-        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
     )
     def __init__(
         self,
         test_particle: particles.Particle,
```

Comment:
Can I still use the `particle_input` decorator? I'm getting issues with the .is_electron component of the tests

Reply:
Yeah, you should be able to - if it doesn't work, leave it as is and we'll fix it elsewhere (just make sure to note it down as a bug!).

Reply:
Yeah I seem to be getting an error when using the decorator there...

Reply:
Ok, that's strange - let's leave it like this for now, then. But if you could reproduce the bug and open an issue with the error message (tagging @namurphy ) that would be great!

Reply:
This worked for me locally - can you update this branch from main and then try this again with ParticleLike? It's possible you just haven't merged main for a while, since the changes to particle_input were like a week ago.